recipes/xlrd/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.0.0" %}
+{% set name = "xlrd" %}
+{% set sha256 = "0ff87dd5d50425084f7219cb6f86bb3eb5aa29063f53d50bf270ed007e941069" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+
+test:
+  imports:
+
+about:
+  home: 'http://www.python-excel.org/'
+  license: "BSD-style"
```

Comment:
The first one is BSD3 and according to what's written in the license this concerns the bulk of the work. The second part of the license concerns earlier work in the library. that license has 4 clauses: the 3 first ones correspond to BSD3 but the fourth item does not correspond to the BSD 4 fourth clause. So, I would think BSD-3 is a reasonable guess. I will add BSD3 for now and add the license file